docker/docker-init.sh

```diff
@@ -37,7 +37,7 @@ Init Step ${1}/${STEP_CNT} [${2}] -- ${3}
 
 EOF
 }
-ADMIN_PASSWORD="admin"
+ADMIN_PASSWORD="datakimia2024"
```

Comment:
I'm not sure if you actually meant to open this PR, but you _probably_ didn't mean to commit this password. You might want to change it on your instance :)